### PR TITLE
Anonymous map user

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: app/src/components/Shared/MenuMobile.jsx [hunk 1]
```diff
@@ -44,9 +44,8 @@ class MenuMobile extends Component {
               href="https://trial-globalfishingwatch.cs43.force.com/gfw/GFWResetPassword"
               className={menuMobile['forgot-password']}
             >
-              Remember password
```
Comment (tiagojsag) [inline app/src/components/Shared/MenuMobile.jsx]: This is the recover link, not an option to remember password on future sessions

File: app/src/components/Shared/MenuMobile.jsx [hunk 2]
```diff
@@ -44,9 +44,8 @@ class MenuMobile extends Component {
...
+              Reset password
             </a>
-            <button className={menuMobile.register} onClick={this.props.login}>Sign up</button>
```
Comment (tiagojsag) [inline app/src/components/Shared/MenuMobile.jsx]: I honestly don't know wth this was doing here

**Comment by andresgnlez** [comment]
Changing basemap is not working. Apart of that, looks good to me.

**Comment by tiagojsag** [comment]
done, ready to test and merge @andresgnlez

**Comment by andresgnlez** [comment]
👍

**Pull request merged**